server/twitch.handler.ts

```diff
@@ -141,7 +138,17 @@ export class TwitchHandler {
         continue;
       }
 
-      if (message.toLowerCase().includes(twitchSetting.contains.toLowerCase())) {
+      if(eventOptions && eventOptions.event === twitchSetting['event']){
+        switch(twitchSetting['event']){
+          case TwitchEventTypes.bits:
+            if(eventOptions.bitsCount<=twitchSetting["bitAmount"]){
+              return twitchSetting;
+            }
+            break;
+        }
+      }
+
+      if (message.toLowerCase().includes(twitchSetting.contains.toLowerCase() || null)) {
```

Comment:
is the ` || null ` really used somewhere? I think it'll never get there :D

Reply:
It is needed once the events are added, because if you don't have a null there it will try to match an empty string  to a message which would return true and trigger all the event clips because of it.

Reply:
OH MY - I didnt receive any of these issues

maybe we could skip twitchSettings which are empty before this check then there wouldn't be a need for `null`